### PR TITLE
add a README to the idm package

### PR DESCRIPTION
**Pull request by C0rby**
I took the description from [owncloud.dev](https://owncloud.dev/services/idm/)

### CODE REVIEW

File: services/idm/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Idm
+
+The IDM service provides a minimal LDAP Service, based on [Libregraph idm](https://github.com/libregraph/idm), for oCIS. It is started as part of the default configuration and serves as a central place for storing user and group information.
+
+It is mainly targeted at small oCIS installations. For larger setups it is recommended to replace IDM with a “real” LDAP server or to switch to an external Identity Management Solution.
+
+IDM listens on port 9325 by default. In the default configuration it only accepts TLS protected connections (LDAPS). The BaseDN of the LDAP tree is o=libregraph-idm. IDM gives LDAP write permissions to a single user (DN: uid=libregraph,ou=sysusers,o=libregraph-idm). Any other authenticated user has read-only access. IDM stores its data in a boltdb file idm/ocis.boltdb inside the oCIS base data directory.
```
Comment (mmattel) [inline services/idm/README.md:7]:
```suggestion
IDM listens on port 9325 by default. In the default configuration it only accepts TLS protected connections (LDAPS). The BaseDN of the LDAP tree is `o=libregraph-idm`. IDM gives LDAP write permissions to a single user (DN: `uid=libregraph,ou=sysusers,o=libregraph-idm`). Any other authenticated user has read-only access. IDM stores its data in a boltdb file `idm/ocis.boltdb` inside the oCIS base data directory.
```

File: services/idm/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+Note: IDM is limited in its functionality. It only supports a subset of the LDAP operations (namely BIND, SEARCH, ADD, MODIFY, DELETE). Also IDM currently does not do any schema verification (e.g. structural vs. auxiliary object classes, require and option attributes, syntax checks, …). So it’s not meant as a general purpose LDAP server.
```
Comment (mmattel) [inline services/idm/README.md:9]:
```suggestion
Note: IDM is limited in its functionality. It only supports a subset of the LDAP operations (namely `BIND`, `SEARCH`, `ADD`, `MODIFY`, `DELETE`). Also, IDM currently does not do any schema verification (like. structural vs. auxiliary object classes, require and option attributes, syntax checks, …). Therefore it is not meant as a general purpose LDAP server.
```
>  Therefore it is not meant as a general purpose LDAP server.

but for...

File: services/idm/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Idm
+
+The IDM service provides a minimal LDAP Service, based on [Libregraph idm](https://github.com/libregraph/idm), for oCIS. It is started as part of the default configuration and serves as a central place for storing user and group information.
+
+It is mainly targeted at small oCIS installations. For larger setups it is recommended to replace IDM with a “real” LDAP server or to switch to an external Identity Management Solution.
```
Comment (EParzefall) [inline services/idm/README.md:5]:
```suggestion
It is mainly targeted at small oCIS installations. For larger setups it is recommended to replace IDM with a “real” LDAP server or to switch to an external identity management solution.
```

File: services/idm/README.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
+IDM listens on port 9325 by default. In the default configuration it only accepts TLS protected connections (LDAPS). The BaseDN of the LDAP tree is `o=libregraph-idm`. IDM gives LDAP write permissions to a single user (DN: `uid=libregraph,ou=sysusers,o=libregraph-idm`). Any other authenticated user has read-only access. IDM stores its data in a boltdb file `idm/ocis.boltdb` inside the oCIS base data directory.
```
Comment (EParzefall) [inline services/idm/README.md:7]:
```suggestion
IDM listens on port 9325 by default. In the default configuration it only accepts TLS-protected connections (LDAPS). The BaseDN of the LDAP tree is `o=libregraph-idm`. IDM gives LDAP write permissions to a single user (DN: `uid=libregraph,ou=sysusers,o=libregraph-idm`). Any other authenticated user has read-only access. IDM stores its data in a boltdb file `idm/ocis.boltdb` inside the oCIS base data directory.
```

**Review by EParzefall — APPROVED** [review]

**Comment by mmattel** [comment]
@C0rby 
>  Therefore it is not meant as a general purpose LDAP server.

but for...?

**Pull request merged**